src/ResourceManager/Network/Commands.Network/PublicIpPrefix/GetAzurePublicIpPrefixCommand.cs

```diff
@@ -39,7 +39,7 @@ public class GetAzurePublicIpPrefixCommand : PublicIpPrefixBaseCmdlet
         public virtual string Name { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
```

Comment:
If you can now supply just name and not ResourceGroupName in the cmdlet call, please add a test for
```
Get-AzureRmPublicIpPrefix -Name <name>
```

Reply:
Currently I updated it similar to [VNet Get cmdlet](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/VirtualNetwork/GetAzureVirtualNetworkCommand.cs#L32) where we have RG and Name both optional while call w/ Name and w/o RG would fail.
In case we keep RG required, ListAll call becomes unreachable (There was no test for ListAll, so this fact was missed).

Alternatively (and more correct) we could update this similar to [NetworkWatcher Get cmdlet](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/NetworkWatcher/GetAzureNetworkWatcherCommand.cs#L39).
However in this case we'll have to make Name mandatory for "Get" param set and remove DefaultParameterSet from Get cmdlet. This is likely would be considered as breaking change.

Reply:
@EvgenyAgafonchikov Changing to the Get-AzureRmNetworkWatcher parameter sets will work fine.  This will not be a breaking change because all of the scenarios supported in the original parameter set will still be supported by the new parameter sets.